resolwe_bio/tests/processes/test_alignment.py

```diff
@@ -307,3 +309,36 @@ def test_subread(self):
         aligned_reads = self.run_process('alignment-subread', inputs)
         self.assertFile(aligned_reads, 'stats', 'subread_paired_reads_report.txt')
         self.assertFile(aligned_reads, 'bam', 'subread_paired.bam')
+
+    @tag_process('alignment-hisat2', 'upload-genome')
+    def test_hisat2_bigwig(self):
+        with self.preparation_stage():
+            paired_reads = self.prepare_paired_reads(mate1=['SRR2124780_1 1k.fastq.gz'],
+                                                     mate2=['SRR2124780_2 1k.fastq.gz'])
+        inputs = {
+            'src': 'hg38_chr21_9M.fa.gz',
+            'species': 'Homo sapiens',
+            'build': 'hg38'
+        }
+        genome_ucsc = self.run_process('upload-genome', inputs)
```

Comment:
Since you don't assert fields of `genome_ucsc`, move this into `self.preparation_stage()` context and remove `upload-genome` from the process tag at the top of this test.